include/wlr/types/wlr_input_timestamps_v1.h

```diff
@@ -9,4 +9,81 @@
 #ifndef WLR_TYPES_WLR_INPUT_TIMESTAMPS_V1_H
 #define WLR_TYPES_WLR_INPUT_TIMESTAMPS_V1_H
 
+#include <wayland-server.h>
+
+/**
+ * This protocol specifies a way for a client to request and receive
+ * high-resolution timestamps for input events.
+ */
```

Comment:
I would put this comment at the top of the file

Reply:
All (except one) headers follow this pattern in this directory, if they have any top level comment. E.g., wlr_cursor, wlr_idle, wlr_idle_inhibit, etc.

Reply:
Yeah, I agree with @arandomhuman here. If we want to move these somewhere else this should be done for all files.